app/helpers/application_helper/button/volume_backup_create.rb

```diff
@@ -0,0 +1,10 @@
+class ApplicationHelper::Button::VolumeBackupCreate < ApplicationHelper::Button::Basic
+  def calculate_properties
```

Comment:
Looks like `ApplicationHelper::Button::GenericFeatureButton` implements exactly the same thing as this button.
And the one below too.

ping @romanblanco , @PanSpagetka